app/server/tests/apiService.test.ts

```diff
@@ -50,13 +50,13 @@ describe("apiService", () => {
         headers: AxiosRequestHeaders,
         transformResponse: AxiosResponseTransformer
     ) => {
-        expect(headers).toStrictEqual({
+        expect(headers.toJSON()).toEqual({
```

Comment:
axios code changed as we updated the package, so just have to explicitly convert to json before checking